docs/sources/datasources/pyroscope/configure-pyroscope-data-source.md

```diff
@@ -42,16 +42,85 @@ To configure basic settings for the data source, complete the following steps:
+
+If you make any changes, select **Save & test** to preserve those changes.
+
+![Configuration options for the Pyroscope data source](/media/docs/grafana/data-sources/screenshot-pyroscope-data-source-config.png)
+
+### Name and default
+
+**Name**
+: Enter a name to specify the data source in panels, queries, and Explore.
+
+**Default**
+: The default data source is pre-selected for new panels.
+
+### HTTP
+
+The HTTP section is shown in number 1 in the screenshot.
+
+**URL**
+: The URL of the Grafana Pyroscope instance, for example, `https://localhost:4100`.
+
+**Allowed cookies**
+: The Grafana Proxy deletes forwarded cookies. Use this field to specify cookies by name that should be forwarded to the data source.
+
+**Timeout**
+: HTTP request timeout in seconds.
+
+### Auth
+
+The Auth section is shown in number 2 in the screenshot.
+
+**Basic auth**
+: Enable basic authentication to the data source. When activated, it provides **User** and **Password** fields.
+
+**With Credentials**
+: Whether credentials, such as cookies or auth headers, should be sent with cross-site requests.
+
+**TLS Client Auth**
+: Toggle on to use client authentication. When enabled, it adds the `Server name`, `Client cert` and `Client key`. The client provides a certificate that is validated by the server to establish the client's trusted identity. The client key encrypts the data between client and server. These details are encrypted and stored in the Grafana database.
+
+**With CA Cert**
+: Activate this option to verify self-signed TLS certificates.
+
+**Skip TLS Verify**
+: When activated, it bypasses TLS certificate verification.
+
+**Forward OAuth Identity**
+: When activated, the user’s upstream OAuth 2.0 identity is forwarded to the data source along with their access token.
+
+**Custom HTTP Headers**
+: Select Add header to add Header and Value fields.
+
+**Header**
+: Add a custom header. This allows custom headers to be passed based on the needs of your Pyroscope instance.
+
+**Value**
+: The value of the header.
+
+### Querying
+
+The **Querying** section is shown in number 3 in the screenshot.
+
+**Minimum step** is used for queries returning time-series data. The default value is 15 seconds.
+
+The Pyroscope backend, similar to Prometheus, scrapes profiles at certain intervals. To prevent querying at smaller interval, use Minimal step same or higher than your Pyroscope scrape interval. This prevents returning too many data points to the frontend.
+
+### Private data source connect
+
+The **Private data source connect** section is shown in number 4 in the screenshot.
+
+This option lets you query data that lives within a secured network without opening the network to inbound traffic from Grafana Cloud.
+
+Use the drop-down box to select a configured private data sources.
+
+Select **Manage private data source connect** to configure and manage any private data sources you have configured.
```

Comment:
Private data source connect is something that Grafana Cloud only. (correct me if I am wrong), so I don't think it should be mentioned here.

Reply:
E.g. Prometheus also doesn't mention this: https://github.com/grafana/grafana/blob/a16782a2d6c5823dd8d1abdc463da9f8d8b8113a/docs/sources/datasources/prometheus/configure-prometheus-data-source.md?plain=1#L112